javascript/hints.js

```diff
@@ -91,7 +91,7 @@ var titles = {
     "Filename word regex": "This regular expression will be used extract words from filename, and they will be joined using the option below into label text used for training. Leave empty to keep filename text as it is.",
     "Filename join string": "This string will be used to join split words into a single line if the option above is enabled.",
 
-    "Quicksettings list": "List of setting names, separated by commas, for settings that should go to the quick access bar at the top, rather than the usual setting tab. See modules/shared.py for setting names. Requires restarting to apply.",
+    "Quick Settings list": "List of setting names, separated by commas, for settings that should go to the quick access bar at the top, rather than the usual setting tab. See modules/shared.py for setting names. Requires restarting to apply.",
```

Comment:
the `key` needs to match both sides will have to be changed

Reply:
Thank you for your valuable suggestion. I sincerely appreciate it and will promptly implement the required adjustments to the code!